app/javascript/mastodon/actions/push_notifications/registerer.js

```diff
@@ -36,7 +36,7 @@ const subscribe = (registration) =>
 const unsubscribe = ({ registration, subscription }) =>
   subscription ? subscription.unsubscribe().then(() => registration) : registration;
 
-const sendSubscriptionToBackend = (getState, subscription) => {
```

Comment:
Why removing `getState`?

Reply:
The Web API can use sessions to identify users. So, I think that OAuth token is unnecessary.

Reply:
I see. I wonder if @Gargron is mentioning this change, or the forgery protection. This could be in a different pull request in such a case. It would also help when this change is to be reverted as Gargron suspects.

Anyway, it is only when my guess is true. @Gargron's confirmation is needed.